app/views/faq_groups/basics.md

```diff
@@ -1,37 +1,36 @@
 # How does Assembly work?
 
-Assembly enables products from software and hardware, to online communities, to be built like open source projects. Contributors earn a part of the product (in App Coins) when they complete a bounty made by the core team of a product.
+Assembly enables products--from software and hardware, to online communities--to be built like open source projects. Contributors earn a part of a product (in App Coins) when they complete a bounty made by that product's core team.
```

Comment:
@lachlanjc Hey there! I didn't use a literal em dash because Markdown parsers will typically convert `--` to an em dash (plus the file didn't reveal a clear convention on this) so it shouldn't matter after the formatting step. Please let me know if I'm missing anything though!